src/client/client.cpp

```diff
@@ -1789,6 +1792,60 @@ float Client::getCurRate()
 			m_con->getLocalStat(con::CUR_DL_RATE));
 }
 
+void Client::storeInventoryImages(u16 res)
+{
+	irr::video::IVideoDriver *driver = RenderingEngine::get_video_driver();
+	infostream<<"- Saving inventory images"<<std::endl;
+	if (driver->queryFeature(video::EVDF_RENDER_TO_TARGET))
+	{
```

Comment:
missing spaces on some operators and braces on if/for are on the same line. Please look at our guidelines or the clang-format file

Reply:
Does https://github.com/minetest/minetest/pull/9727/commits/058a31a987f224bc47e512cfea80d3c4e89cf412 fix it?

Also, I'm a bit confused, I thought braces on if/for should be on the next line?

Reply:
Opening braces should be on the same line, as in your latest commit which looks fine.
Unfortunately MT code still has many places with bad codestyle, so can be misleading.
Codestyle guidelines are in the developer wiki, apart from that, we roughly follow linux kernel codestyle.

Reply:
Thanks for the clarification.